omod/src/main/webapp/uploadForm.jsp

```diff
@@ -162,7 +162,7 @@ h4 {
 						</td>
 					</tr>
 					<tr>
-						<td>Report Name*</td>
+						<td>'<spring:message code="birt.reportName"/>'</td>

```

Comment:
@robbyoconnor there are free-texts with quotation marks and without. So, as i think it's different. I read that it's a good practice to use a different style of quotation marks when there are 2 quotes...